joint_calling/upload_processor.py

```diff
@@ -67,11 +72,23 @@ def batch_move_files(
         " """
 
     jobs = []
-    for sample in files:
-        previous_location = os.path.join('gs://', source_prefix, sample)
-        new_location = os.path.join('gs://', destination_prefix, sample)
 
-        j = batch.new_job(name=f'move {sample}')
+    external_id = sample_group.sample_id_external
+    internal_id = sample_group.sample_id_internal
+    batch_number = sample_group.batch_number
+
+    for tuple_key in sample_group._fields:
+        if tuple_key in ['sample_id_external', 'batch_number', 'sample_id_internal']:
+            continue
+        file_name = getattr(sample_group, tuple_key)
```

Comment:
It'd be clearer to define what files are being moved, rather than having to refer back to the sample_group definition to determine.

```suggestion
    files_to_move = (sample_group.data_file, sample_group.index_file, sample_group.md5)
    for file_name in files_to_move:
```